src/app/src/components/FacilitiesMapErrorMessage.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+const mapErrorMessageStyles = Object.freeze({
+    errorMessageContainerStyles: Object.freeze({
+        height: '100%',
+        width: '100%',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+    }),
+});
+
+export default function FacilitiesMapErrorMessage() {
```

Comment:
I did not memoize this because doing so issues a linter error from the `Route component` call in which it is instantiated which I think comes from a quirk with how our version of react-router sees memoized components.